pages/references.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">Project References</h1>
+    <p class="mb-3">
+      <span class="font-medium">Database Source:</span> Julian Goodare, Lauren
+      Martin, Joyce Miller and Louise Yeoman,
+      <a
+        class="underline hover:text-gray-500"
+        href="http://www.shca.ed.ac.uk/witches/"
+        target="_blank"
+      >
+        'The Survey of Scottish Witchcraft' </a
+      >(archived January 2003)
+    </p>
+    <br />
+    <h2 class="mb-3">Place-Name References</h2>
+    <p class="mb-3">
+      Alexander, W.M.C. & Third Spalding Club (1952). The place-names of
+      Aberdeenshire, Aberdeen: Printed for the Third Spalding Club.
+    </p>
+    <p class="mb-3">Blaeu, J (1654). Blaeu's Atlas novus, volume V.</p>
+    <p class="mb-3">
+      Johnston, J.B. & Royal Scottish Geographical Society (1940). The
+      place-names of Berwickshire, Edinburgh: Royal Scottish Geographical
+      Society.
+    </p>
+    <p class="mb-3">
+      Márkus, G (2012). The place-names of Bute, Donington [Lincolnshire]: Shaun
+      Tyas.
+    </p>
+    <p class="mb-3">
+      National Library of Scotland (2019). Explore georeferenced maps - Map
+      images - National Library of Scotland. [online] Available at:
+      <a
+        class="underline hover:text-gray-500"
+        href="https://maps.nls.uk/geo/explore/#zoom=5&lat=56.0000&lon=-4.0000&layers=1&b=1"
+        >https://maps.nls.uk/geo/explore/#zoom=5&lat=56.0000&lon=-4.0000&layers=1&b=1
+      </a>
+      [Accessed 16 Aug. 2019].
+    </p>
+    <p class="mb-3">
+      Ordnance Survey (1885). Map of Great Britain, One Inch 1885-1903. London,
+      Ordnance Survey.
+    </p>
+    <p class="mb-3">
+      Ordnance Survey (1888). Map of Great Britain, Six Inch 1888-1913. London,
+      Ordnance Survey.
+    </p>
+    <p class="mb-3">
+      Ordnance Survey (1892). Map of Great Britain, Twenty Five Inch 1892-1914.
+      London, Ordnance Survey.
+    </p>
+    <p class="mb-3">Pont, T (1583-1614). Pont Maps of Scotland.</p>
+    <p class="mb-3">
+      Scottish-Places (2019). The Gazetteer for Scotland. [online] Available at:
+      <a
+        class="underline hover:text-gray-500"
+        href="https://www.scottish-places.info/"
+      >
+        https://www.scottish-places.info/ </a
+      >[Accessed 16 Aug. 2019].
+    </p>
+    <p class="mb-3">
+      ScotlandsPlaces (2019). ScotlandsPlaces. [online] Available at:
+      <a
+        class="underline hover:text-gray-500"
+        href="https://scotlandsplaces.gov.uk/"
+      >
+        https://scotlandsplaces.gov.uk/
+      </a>
+      [Accessed 16 Aug. 2019].
+    </p>
+    <p class="mb-3">
+      Statistical Accounts (2019). Statistical Accounts of Scotland. [online]
+      Available at:<a
```

Comment:
Is there a way to apply a global style to the links? Seems a  waste of effort for us to apply styles individually to every link. Thanks Ruby

Reply:
done in layouts

Reply:
actually had to remove this as was affecting the navigation bar links, instead i created a link class in the layouts that i have added to the links instead of using tailwind for each

Reply:
Cool, thanks Ruby.